src/v2/client/startLoginFlow.js

```diff
@@ -43,19 +43,22 @@ export async function startLoginFlow(settings) {
   const stateHandleFromSession = sessionStorageHelper.getStateHandle();
 
   if (stateHandleFromSession) {
+    // try to introspect with stateHandle from sessionStorage
```

Comment:
we don't need to repeat what code is doing in comments.. probably focus on `why?` while writing comments..